pkg/distro/rhel9/images.go

```diff
@@ -199,9 +199,18 @@ func osCustomizations(
 				panic(fmt.Sprintf("unexpected error creating tailoring file options: %v", err))
 			}
 
+			var overrides []osbuild.OscapAutotailorOverride
+			for _, override := range oscapConfig.Tailoring.Overrides {
+				overrides = append(overrides, osbuild.OscapAutotailorOverride{
+					Var:   override.Var,
+					Value: override.Value,
+				})
+			}
```

Comment:
Seeing this multiple times in each distro, it feels like a common conversion function would make sense, instead of duplicating the logic. Basically something to convert `oscapConfig.Tailoring` to `osbuild.OscapAutotailorConfig`. What do you think? :thinking:

Reply:
Yeah, that's a good idea. I will do that :smile:

Reply:
I did quite a big refactor here, but in the long run I think it's worth it. Let me know what you think

Reply:
I had a few comments, but in general I like it a lot. Nice work!